[Documentation] Update documentation regarding ReCaptcha in graphQL and REST requests (#283)
AdobeDocs/commerce-webapi: src/pages/graphql/usage/protected-mutations.md

# Protected mutations

> If CAPTCHA or reCAPTCHA is enabled on pages requiring shopper input, then in most cases, the corresponding mutations that send requests to the application server must include an HTTP header that contains a value entered by the shopper (for CAPTCHA) or generated by the Google API (for reCAPTCHA). These headers are generated by UI Web form widgets during interacting with shopper. However, if you specify an integration authorization token in the header of the mutation, then you do not supply a header specific to CAPTCHA or reCAPTCHA.

Reviewer: ```suggestion
If CAPTCHA or reCAPTCHA is enabled on pages requiring shopper input, then in most cases, the corresponding mutations that send requests to the application server must include an HTTP header that contains a value entered by the shopper (for CAPTCHA) or generated by the Google API (for reCAPTCHA). These headers are generated by UI Web form widgets during interactions with the shopper. However, if you specify an integration authorization token in the header of the mutation, then you do not supply a header specific to CAPTCHA or reCAPTCHA.
```

> Even when Adobe Commerce is used in a headless way, like with PWA, the Merchant must have their own Web UI which must include rendering the correct Captcha or reCAPTCHA web form widget, to capture the proper `X-Captcha` or `X-ReCaptcha` HTTP header, and send it in the API request in the background only.

Reviewer: ```suggestion
Even when Adobe Commerce is used in a headless way, like with PWA, the merchant must have their own Web UI that renders the correct Captcha or reCAPTCHA web form widget, captures the proper `X-Captcha` or `X-ReCaptcha` HTTP header, and sends it in the API request in the background only.
```
